src/llmcompressor/args/dataset_arguments.py

```diff
@@ -217,3 +217,6 @@ class DatasetArguments(CustomDatasetArguments):
             "Default is set to True."
         },
     )
+
+    def is_dataset_required(self) -> bool:
```

Comment:
This seem less like a `is_dataset_required` and more like `is_dataset_provided`. Is the name wrong or the function?

Reply:
Or maybe it should be `is_processor_required`?

Reply:
updated